ci/aws-iam-check-keys/find_stale_keys.py

```diff
@@ -352,11 +381,11 @@ def check_key(
     }
     print(f"user is either being sent or deleted: {user_dict}")
     if days_since_rotation >= user.violation and user.account_type:
-        send_key(user_dict, "violation")
+        send_key(user_dict, "violation", False)
     elif days_since_rotation >= user.warn:
-        send_key(user_dict, "warn")
+        send_key(user_dict, "warn", False)
     else:
-        del_key(user_dict)
+        send_key(user_dict, None, True)
```

Comment:
Are you ready for this to delete or still testing?

Reply:
It's just deleting a metric in staging prometheus, and yes I'm ready for deleting, but I'm still testing as well